spell-wordlist-fr.txt

```diff
@@ -0,0 +1,156 @@
+Accessibility
```

Comment:
(I'll just put a single comment here for all the words I question instead of adding a comment for each)

There is a couple of words that we should try to remove from this file _if we can_.  (Feel free to question these, in most cases I'm fairly certain but am not the best at French :))

- "decision": Should be "décision" in the text instead  (right?)
- "mega": Should it be "méga"?  (I've seen both "mega" and "méga" in french on the internet but if the spellcheck accepts "méga" we should stick to that.
- "selection": Should be "sélection" in the text instead (right?)
- `tree`:   That one is in an `alt` attribute in file `index-fr.html#90` that is still in english.  The `alt` attribute text should be translated instead.